x-pack/metricbeat/module/azure/monitor_service.go

```diff
@@ -43,9 +43,10 @@ type MonitorService struct {
 }
 
 const (
-	metricNameLimit        = 20
-	ApiVersion             = "2021-04-01"
-	BatchApiResourcesLimit = 50
+	metricNameLimit          = 20
+	ApiVersion               = "2021-04-01"
+	BatchApiResourcesLimit   = 50
+	BatchApiMetricNamesLimit = 20
```

Comment:
It seems the non-batch version has he same limit (the non-batch API uses `metricNameLimit` to limit the number of metric names per request).

Reply:
Indeed! I used the same global var.